.changeset/shiny-eagles-stare.md

```diff
@@ -0,0 +1,6 @@
+---
+'@directus/extensions-sdk': patch
+'@directus/api': patch
```

Comment:
I believe this is considered a breaking change as previously it would always be set to production and now in the isolate it is whatever is defined (or nothing). cc @licitdev 

```suggestion
'@directus/api': major
```

Reply:
Yes, I could see this being a breaking change. I think the `extensions-sdk` should also be marked as a breaking change. What do you think?